openassessment/xblock/static/sass/oa/views/_oa-course-items-listing.scss

```diff
@@ -64,22 +64,23 @@
   }
 
   .ora-summary-value {
-    font-size: 2.2em;
+    font-size: 1.5em;
     padding-top: 3px;
+    text-overflow: ellipsis;
```

Comment:
Nit: this css class and the `backgrid` one share same `text-overflow` and `overflow` attributes and it could b pulled into a placeholder selector (e.g. [%ui-box in Insights](https://github.com/edx/edx-analytics-dashboard/blob/aec067b1dddf7bfe10299163b720578fa2cf622a/analytics_dashboard/static/sass/_base.scss#L73) and ["extended" in an actual class](https://github.com/edx/edx-analytics-dashboard/blob/aec067b1dddf7bfe10299163b720578fa2cf622a/analytics_dashboard/static/sass/_base.scss#L355).